src/rules/function-calc-no-unspaced-operator/index.js

```diff
@@ -23,6 +23,8 @@ export default function (actual) {
     if (!validOptions) { return }
 
     root.walkDecls(decl => {
+      if (decl.value.indexOf("calc") !== 0) { return }
+
```

Comment:
@davidtheclark maybe use postcss-value-parser? Also need add this example to test.